docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+version: "2.1"
+services:
+  opbeans-java:
+    build: .
+    image: opbeans/opbeans-java:latest
+    ports:
+      - "127.0.0.1:${OPBEANS_SERVER_PORT:-8000}:8000"
+    logging:
+      driver: 'json-file'
+      options:
+          max-size: '2m'
+          max-file: '5'
+    environment:
+      - ELASTIC_APM_SERVICE_NAME=${ELASTIC_APM_SERVICE_NAME:-opbeans-java}
```

Comment:
Seems like `OPBEANS_JAVA_PROFILE` and maybe also`DATABASE_URL`, `DATABASE_DRIVER` and `DATABASE_DIALECT` are missing as you have removed the default values?

Reply:
I've removed the default from the docker file, but not from the opbeans application: 
https://github.com/elastic/opbeans-java/blob/master/opbeans/src/main/resources/application.properties#L13-L16

I didn't like that the dockerfile was always overriding the application defaults and in this case for the docker-compose I wanted to use H2 and make it simple.